Spellcheckmain-errordetection.py

```diff
@@ -0,0 +1,48 @@
+#SpellChecker

```

Comment:
Great! This entire program handles the processing of the words and returning the wrong list of words.